### ISSUE TITLE
Unable to login to dashboard

### ISSUE THREAD

**Issue by tkdave** [ant-media/Ant-Media-Server#3220]
### Short description
Cluster is not letting us login today. Using latest AMI.

### Environment

* Operating system and version: Ubuntu 18
* Ant Media Server version: 2.3.2? (latest AMI)
* Browser name and version: Chrome, Brave, others

### Steps to reproduce
1. Login with existing username and password - says username or password is incorrect
2. Username was added to MongoDB using a script that follows directions found in FAQ
3. Confirmed that user record is present in Mongo

### Expected behavior
Should be able to login.

### Actual behavior
Cannot get into dashboard.

### Logs
No errors in server logs.Mongo connection is successful from origin.

One console error: 
Call to `/rest/isEnterpriseEdition` is failing with 403. Not sure if that is expected on login page or not.

**Comment by SelimEmre** [comment]
Hi @tkdave,
We are storing passwords with MD5 encryption in the latest version. You need to change your passwords to MD5 encryption. You can encrypt your password basically any MD5 encrypter page like:  https://www.md5online.org/md5-encrypt.html

Here are the steps:
1- Login MongoDB
2- Run below commands:
`use serverdb`
`db.getCollection('User').find() //Get User details`
`db.User.updateOne({"_id": "userID"}, {$set:{password: "md5Password"}}) //Use your User ID in updateOne section and use password with MD5 protection.`

**Comment by mekya** [comment]
Hi @tkdave ,

Is your problem resolved with @SelimEmre suggestion?

If yes, I'm going to close the issue.

**Comment by tkdave** [comment]
Yes. This is resolved. 

For reference, more discussion is on the google group here:
https://groups.google.com/u/1/g/ant-media-server/c/G2MmQEkN3L0

And, we have updated the FAQ with MD5 note here:
https://github.com/ant-media/Ant-Media-Server/wiki/Frequently-Asked-Questions#how-to-reset-admin-password

I updated my cluster launch script to include the MD5 version of the password, and all is well. Thanks for the follow-up.
